### PR TITLE
RO-4246 add job for mnaio master-rc

### PR DESCRIPTION
**Pull request by zreichert**
Issue: [RO-4246](https://rpc-openstack.atlassian.net/browse/RO-4246)

### CODE REVIEW

File: rpc_jobs/rpc_openstack.yml [hunk 1]
```diff
@@ -234,6 +234,32 @@
...
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branch: "master-rc"
+    jira_project_key: "RO"
+    image:
+         - xenial_mnaio_no_artifacts:
+             FLAVOR: "onmetal-io2"
+             IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+             REGIONS: "IAD"
+             FALLBACK_REGIONS: "DFW"
+       scenario:
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: Hey @zreichert , the indentation here looks a bit funky.

Comment (zreichert) [reply rpc_jobs/rpc_openstack.yml]: Yeah thats not right.  I'll push a fix.

Comment (zreichert) [reply rpc_jobs/rpc_openstack.yml]: @mattt416 Have a look now

**Comment by mattt416** [comment]
@zreichert Please squash these 2 commits down when this one merges. Thanks!

**Comment by chundu** [comment]
@zreichert Looks this PR is good to merge, still waiting for something?

**Pull request merged**